src/components/GameBoard.tsx

```diff
@@ -64,7 +64,8 @@ function GameBoard() {
         <PerspectiveCamera makeDefault position={[0, 0, cameraZoom]} />
         {orbitControls && <OrbitControls />}
         <Croupier />
-        <ExtinctionTiles />
+
+        <ExtinctionTiles gameState={state} />
```

Comment:
we have a hook to get game state from context https://github.com/helcomsecretariat/ecosfera-baltica-digital/blob/d72cca34f7136103b892cb0f75b20b24ef5b6b4f/src/components/Croupier.tsx#L29

Reply:
Cool thank you forgot about that

Reply:
Changed it